README.md

```diff
@@ -25,11 +25,14 @@ https://github.com/acheong08/ChatGPT/wiki/Setup
 ```json
 {
   "session_token": "<token>",
-  "proxy": "<proxy>"
+  "proxy": "<proxy>",
+  "driver_exec_path": "./path/to/driver",
+  "browser_exec_path": "./path/to/browser",
+  "conversation": "00000000-0000-0000-0000-000000000000"
```

Comment:
This is optional I hope? Please put them in wiki instead of readme and ensure that it is marked as optional

Reply:
I did indicate that they are optional but I will move to wiki

Reply:
Just make it a bit more clear